src/components/Match/Match.js

```diff
@@ -20,8 +21,9 @@ type Props = {
   handleSaveOrDiscardPress: ( ) => void,
   navToTaxonDetails: ( ) => void,
   handleLocationPickerPressed: ( ) => void,
-  suggestions: Object,
-  onTaxonChosen: ( ) => void
+  topSuggestion: Object,
+  otherSuggestions: Array<Object>,
```

Comment:
This file is not using flow and is not typescript, so these types probably shouldn't be here. If you do decide to convert to TypeScript, using `Object` types doesn't achieve the goal of allowing IDEs to provide developers with hints about how to use the component (which I think is why we decided to write new code in TS in the first place, right?), nor does it provide much type safety. IMO it's totally fine to use JS for speed, but I don't think incomplete typing like this helps.

Reply:
Yeah, I see that. Total habit after 6 years of using... mostly Flow in this way. I think I'm going to leave for now because this warrants a bigger audit of all the files I've written in the past few weeks. I'm also sure we need to have a team convo soon about TS vs. Flow vs. no typing.